pyamg/aggregation/smooth.py

```diff
@@ -539,6 +590,7 @@ def cgnr_prolongation_smoothing(A, T, B, BtBinv, Sparsity_Pattern, maxiter,
                            Sparsity_Pattern.indptr),
                           shape=(Sparsity_Pattern.shape))
     AT = -1.0*A*T
+    cost[0] +=  T.nnz / float(T.shape[0])
```

Comment:
Think this is actually a remnant of the estimate we originally used for mat-mat multiplies, namely that FLOPs are approximately T.nnz * A.nnz / num_cols(T). Then to get WUs, we divide by A.nnz, and it gives T.nnz / num_cols(T). I didn't account for the -1 because in a high performance setting I know there are tricks that make that really fast.